improver/utilities/spatial.py

```diff
@@ -169,11 +169,30 @@ class DifferenceBetweenAdjacentGridSquares(BasePlugin):
     individually.
     """
 
-    def __init__(self, gradient=False):
+    def __init__(self):
         """
         Initialise class.
         """
-        self.is_gradient = gradient
+        pass
+
+    def _update_metadata(self, diff_cube, coord_name, cube_name):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5583761)